src/block/utils/FallableTrait.php

```diff
@@ -54,4 +54,11 @@ public function onNearbyBlockChange() : void{
 			$fall->spawnToAll();
 		}
 	}
+
+	public function onHitGround(FallingBlock $blockEntity) : ?Block{
+		if($this instanceof Block){
```

Comment:
```suggestion
		if(!$this instanceof Block){
```
I really don't like this assertion though...